crates/networking/rpc/rpc.rs

```diff
@@ -170,6 +173,7 @@ pub async fn start_api(
         active_filters: active_filters.clone(),
         syncer: Arc::new(syncer),
         client_version,
+        peer_handler,
```

Comment:
Could we refactor this to group a bunch of p2p related concepts like `local_p2p_node`,  `local_node_record `, `peer_handler `,  maybe `syncer ` into something like `Node` or similar? Maybe out of scope of this PR, but `RpcApiContext ` is like a bag were we just throw random things, would be nice to be a bit more thoughtful in this regard.

Reply:
PR opened https://github.com/lambdaclass/ethrex/pull/2752

Reply:
No?

Reply:
`NodeData` contains static data about our node, while the PeerHandler actually has its own behaviour and has nothing to do with the data stored in `NodeData`, that's why Id rather keep them separate